src/Parser/Druid/Balance/Modules/Core/GlobalCooldown.js

```diff
@@ -0,0 +1,31 @@
+import SPELLS from 'common/SPELLS';
+import CoreGlobalCooldown from 'Parser/Core/Modules/GlobalCooldown';
+import Channeling from 'Parser/Core/Modules/Channeling';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import Abilities from '../Features/Abilities';
+import Haste from './Haste';
+
+/**
+ * The talent Starlord reduces GCD and cast time of empowered Lunar Strikes and Solar Wraths by 20%. 
+ * Since Solar Wrath cast time == GCD the GCD needs to be reduced.
+ */
+class GlobalCooldown extends CoreGlobalCooldown {
+  static dependencies = {
+    abilities: Abilities,
+    haste: Haste,
+    combatants: Combatants,
+    channeling: Channeling,
+  };
+
+  getCurrentGlobalCooldown(spellId = null) {
```

Comment:
Try to avoid copying any code, especially from core classes as they may change at any time. You can access the base method by using `super.getCurrentGlobalCooldown(spellId)`, if you put this in a var you can don't need to do the GCD calculating twice.

Reply:
It uses super now